content/posts/android-next.md

```diff
@@ -0,0 +1,42 @@
+---
+title: 'Android Developers Rejoice'
+subtitle: 'We have a brand new web wrapper for Android sans Cordova!'
+author: 'dkanada'
+githubusername: 'dkanada'
+date: 2020-08-28T04:16:09-04:00
+justify: 'center'
+---
+
+This will be a net benefit for users and developers alike!
+Everyone will enjoy the native video player spearheaded by [Stampede10343](https://github.com/Stampede10343) and [vitorsemeano](https://github.com/vitorsemeano) over the course of several months.
```

Comment:
Yes we have the new video player but it's not enabled by default. Should be said more clearly IMO

Reply:
I now see it's mentioned in the ExoPlayer section